prometheus-rules/prometheus-vmware-rules/templates/prometheus-alerts.yaml

```diff
@@ -1,4 +1,12 @@
 {{- $root := . }}
+{{- $bedrockAlerts := .Values.bedrockAlerts }}
+{{- $filteredBedrockAlerts := dict }}
```

Comment:
Can you add an example in the `values.yaml`  showing this behaviour, can be empty or commented out.